front/app/containers/ProjectsShowPage/index.tsx

```diff
@@ -88,32 +81,28 @@ const ProjectsShowPage = ({ project }: Props) => {
     sort: '-start_at',
   });
 
-  const loading = useMemo(() => {
-    return anyIsUndefined(locale, appConfig, project, phases?.data, events);
-  }, [locale, appConfig, project, phases, events]);
-
   // Check that all child components are mounted
   useEffect(() => {
     setMounted(true);
   }, []);
 
   // UseEffect to scroll to event when provided
   useEffect(() => {
-    if (scrollToStatusModule && mounted && !loading) {
```

Comment:
I went to the idea page, mimicking a rather small screen on a slow 4G connection, and even then, the scroll still works. So safe to remove, I'd say.

Reply:
I think it depends on the # of ideas, # events, etc. If we're only testing a small # of ideas/events I think even on 4G it's probably fast. 

I believe the issue occurred when there were more :thinking: